### ISSUE TITLE
refactor `oras push` to use handlers for command output

### ISSUE THREAD

**Issue by shizhMSFT** [oras-project/oras#1549]

**Comment by TerryHowe** [comment]
Was this supposed to be for `oras blob push` ? I think that is the only push missing handlers.

**Comment by wangxiaoxuan273** [comment]
> Was this supposed to be for `oras blob push` ? I think that is the only push missing handlers.

Yeah it looks like there's no more work to do for handler refactoring on `oras push`.
